### ISSUE TITLE
Command run with MS app return an error

### ISSUE THREAD

**Issue by LittleVaaty** [LittleVaaty/VDesk#9]
Trying to run this command. Outlook opens but on the same desktop as I'm running the command on, Desktop 3.

C:\temp\Downloads>VDesk run --no-switch --on 1 "C:\Program Files\Microsoft Office\root\Office16\OUTLOOK.EXE"
<3>VDesk.Commands.RunCommand[0] Element not found. (0x8002802B (TYPE_E_ELEMENTNOTFOUND))
Windows version: 10.0.22621.3447

**Comment by cruzer619** [comment]
> Ok good, i will close this issue and open a new one for MS App See #9

also I did add its path to my environmental variables the error message went away.. but still didn't run app in desktop 1

**Comment by DoS007** [comment]
Probably has something to do with [this Process.Start from System.Diagnostics](https://github.com/LittleVaaty/VDesk/blob/2cff91a709f15a74be1f46e61bb325084a0b1ba9/VDesk/Services/ProcessService.cs#L38), relating [doc is here](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.process.start?view=net-8.0).

Maybe [this ](https://stackoverflow.com/questions/73136244/using-c-sharp-process-startmsedge-app-https-www-google-com-opens)or [that ](https://stackoverflow.com/questions/42521332/launching-a-windows-10-store-app-from-c-sharp-executable)migh help. So you could try:
`VDesk run --no-switch --on 1 "C:\Program Files\Microsoft Office\root\Office16\OUTLOOK.EXE" -a "--app"` or
`VDesk run --no-switch --on 1 "shell:AppsFolder\4DF9E0F8.Netflix_mcm4njqhnhss8!Netflix.App"` where Netflix should be replaces with the correct text for Outlook.

**Comment by LittleVaaty** [comment]
After some testing it aper that the app is to slow to start and when vdesk try to move the window in the correct desktop, the window does't exit.
I will add a parameter to force vdesk to wait the choosen amount of time after the start of the process

**Comment by cruzer619** [comment]
going to assume -w switch will work on next revision.. as current vdesk doesn't recognize the switch.

**Comment by cruzer619** [comment]
just installed latest release..  and no-switch is not working.  ran on desktop 3 and outlook runs on desktop 3 instead of desktop 1. Removing the no-switch does run on desktop 1.

vdesk run --on 1 --no-switch -w 1000 "C:\Program Files\Microsoft Office\root\Office16\OUTLOOK.EXE"

**Comment by LittleVaaty** [comment]
Yeah i know the error. It's related to the way vdesk handle process.
I'm currently working on rewrite this part, but it will take me some time to figure it out.
